### ISSUE TITLE
Black player on playback

### ISSUE THREAD

**Issue by rankorn** [piemonte/PBJVideoPlayer#57]
I have a problem where sometimes (around 50%) I get a black playback view after composing a video and trying to show it with PBJVideoPlayer.
My code is very similar to the example:

```
    _videoPlayerController = [[PBJVideoPlayerController alloc] init];
    _videoPlayerController.delegate = self;
    _videoPlayerController.view.frame = _playerView.bounds;
    [self addChildViewController:_videoPlayerController];
    [_playerView  addSubview:_videoPlayerController.view];
    [_videoPlayerController didMoveToParentViewController:self];

    _videoPlayerController.videoPath = [self finalVideoPath];
```

The video path if not nil for sure, and the bounds are correct.

I guess it is some kind of racing condition problem, this code is executed on a compose video finish event I have, but I am not sure what can be done to refresh the player or something similar.

Any ideas?

**Comment by rankorn** [comment]
Seems to be an iOS 10 bug which is due to the use of AVVideoCompositionCoreAnimationTool

**Comment by polihym** [comment]
Also I get the black layer sometimes when playing video with local saved filepath though. Did you find a good way for that?

**Comment by rankorn** [comment]
I just removed a layer in my video composition that included a watermark. I did this only if the device runs iOS 10.
Maybe this is fixed on 10.1, I am not sure..

**Comment by polihym** [comment]
Thanks for your kind reply.
Yes, I think so. but..  there is no any solution for that problem really?
